clisops/core/subset.py

```diff
@@ -1062,11 +1074,16 @@ def subset_shape(
         try:
             # Extract CF-compliant CRS_WKT from crs variable.
             raster_crs = CRS.from_cf(ds_copy.crs.attrs)
+
         except AttributeError:
             raster_crs = False
-    if (raster_crs is False) or (shape_crs is False):
-        warnings.warn(
-            "`raster_crs` and/or `shape_crs` were not given directly or through `ds` and `shape`, no check of compatibility will be perform."
+            if (np.min(lon) >= 0 and np.max(lon) <= 360) and not (
+                np.min(lon) >= -180 and np.max(lon) <= 180
+            ):
+                wrap_lons = True
+    if raster_crs is False:
+        logger.warning(
+            "`raster_crs` was not given directly or through `ds`, no check of compatibility with `shape_crs` will be perform."
         )
     else:
         _check_crs_compatibility(shape_crs=shape_crs, raster_crs=raster_crs)
```

Comment:
Sorry, me again being opiniated.

I think  `raster_crs` could be completely removed from this function. The following suggestion aimes to replace line 1068 to 1089, but github doesn't allow me:
```python3
           if (np.min(lon) >= 0 and np.max(lon) <= 360) and not (np.min(lon) >= -180 and np.max(lon) <= 180 ):
                wrap_lons = True
```
I.e. : Only the wrapping of lons counts and it can be fully determined from the `lon` variable. The actual raster's crs has no impact on this. There are no needs to have a compatibility between the shape's crs (that is now forced to epsg:4326 anyway) and the raster's.

Reply:
so remove even the arg `raster_crs`?

Reply:
I don't understand why wrap_lon was only used in the context where `raster_crs` is not given, but actually `raster_crs` was never used. (even before this PR)

Reply:
I think it was assumed that if the data had a defined `crs` than the longitude variable did not need wrapping. Probably, this is true most of the time as the most common convention for longitude is -180, 180.

And yes, my suggestion is to remove  `raster_crs` as an argument.